.github/workflows/post-publish-test.yml

```diff
@@ -0,0 +1,45 @@
+# This action is an integration test for setup-jfrog-cli workflow
```

Comment:
Does this actually tests that a build info has been published?
Because we are not failing the build if auto publish has failed, maybe we should test this elsewhere. 
Or this file was for developing purposes